### PR TITLE
[refactor][style] fix compile warnings, missing return statements, etc.

### PR DESCRIPTION
**Pull request by steakhal**

### CODE REVIEW

File: analyzer/tools/build-logger/src/ldlogger-util.c [hunk 1]
```diff
@@ -708,5 +706,5 @@ int logPrint(char* logLevel_, char* fileName_, int line_, char *fmt_,...)
   fclose(stream);
   freeLock(lockFd);
 
-  return 0;
+  return;
```
Comment (csordasmarton) [inline analyzer/tools/build-logger/src/ldlogger-util.c:709]: There is no need for this return statement here:
```suggestion
```

**Review by csordasmarton — COMMENTED** [review]
One tiny comment otherwise LGTM!

File: analyzer/tools/build-logger/src/ldlogger-util.c (hunk 1 above)
Comment (steakhal) [reply analyzer/tools/build-logger/src/ldlogger-util.c]: I'm not even sure how I missed that xD

**Review by csordasmarton — APPROVED** [review]
LGTM!

**Review by martong — COMMENTED** [review]
LGTM

**Pull request merged**